### PR TITLE
Add integration tests for API Logging feature

### PR DESCRIPTION
**Pull request by binodmx**
Resolves https://github.com/wso2/product-apim-tooling/issues/867

### CODE REVIEW

File: import-export-cli/integration/apiLogging_test.go [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+import (
+	"testing"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/apim"
+	"github.com/wso2/product-apim-tooling/import-export-cli/integration/testutils"
+)
+
+// Get log levels of APIs of the carbon.super tenant in an environment as a super admin user
+func TestGetAPILogLevelsSuperAdminUser(t *testing.T) {
```
Comment (Chamindu36) [inline import-export-cli/integration/apiLogging_test.go:29]: @binodmx Shall we write these tests in the Table driven format[1], so the code length will be reduced and all user scenarios can be covered (Super tenant Admin, Tenant Admin, Super tenant User and Tenant user)?

[1]. https://github.com/wso2/product-apim-tooling/blob/c8134c2b5948fe4b9edd957696800ef713cb0372/import-export-cli/integration/apiProduct_test.go#L1267-L1322

File: import-export-cli/integration/apim/apim.go [hunk 2]
```diff
@@ -116,6 +118,7 @@ func (instance *Client) Setup(envName string, host string, offset int, dcrVersio
 	instance.devPortalRestURL = getDevPortalRestURL(host, offset, restAPIVersion)
 	instance.publisherRestURL = getPublisherRestURL(host, offset, restAPIVersion)
 	instance.adminRestURL = getAdminRestURL(host, offset, restAPIVersion)
+	instance.devopsRestURL = getDevOpsRestURL(host, offset, "v1")
```
Comment (Chamindu36) [inline import-export-cli/integration/apim/apim.go:121]: Shall we use a constant here?

File: import-export-cli/integration/apim/apim.go [hunk 3]
```diff
@@ -1967,21 +2038,12 @@ func getAdminRestURL(host string, offset int, version string) string {
 	return "https://" + host + ":" + strconv.Itoa(port) + "/api/am/admin/" + version
 }
 
-func getTokenURL(host string, offset int) string {
+func getDevOpsRestURL(host string, offset int, version string) string {
```
Comment (wasuradananjith) [inline import-export-cli/integration/apim/apim.go:2041]:
```suggestion
func getDevOpsRestURL(version, host string, offset int) string {
```
Shall we refactor the method signature by grouping the string arguments together, as shown above, to reduce the number of characters per line?

**Comment by wasuradananjith** [comment]
Seems we have not generated docs for this feature in [1]. Shall we generate the docs as shown in [2] and commit those files to this PR? (After generating the docs there will be changes apictl_set.md [3] file. Make sure **not to commit** those changes)

[1] https://github.com/wso2/product-apim-tooling/pull/865
[2] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/README.md?plain=1#L35
[3] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/docs/apictl_set.md

File: import-export-cli/integration/apim/apim.go (hunk 2 above)
Comment (binodmx) [reply import-export-cli/integration/apim/apim.go]: retrieved the version from config.yaml

File: import-export-cli/integration/apim/apim.go (hunk 3 above)
Comment (binodmx) [reply import-export-cli/integration/apim/apim.go]: Changed all the get rest url methods with requested change.

**Comment by binodmx** [comment]
> Seems we have not generated docs for this feature in [1]. Shall we generate the docs as shown in [2] and commit those files to this PR? (After generating the docs there will be changes apictl_set.md [3] file. Make sure **not to commit** those changes)
> 
> [1] #865 [2] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/README.md?plain=1#L35 [3] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/docs/apictl_set.md

Added docs.

**Comment by binodmx** [comment]
> Seems we have not generated docs for this feature in [1]. Shall we generate the docs as shown in [2] and commit those files to this PR? (After generating the docs there will be changes apictl_set.md [3] file. Make sure **not to commit** those changes)
> 
> [1] #865 [2] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/README.md?plain=1#L35 [3] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/docs/apictl_set.md

Added docs.

**Review by wasuradananjith — APPROVED** [review]

**Pull request merged**